### PR TITLE
docs: correct, edit, reformat, otherwise improve API infos

### PR DESCRIPTION
**Pull request by j9t**
Resolves DX-266

Signed-off-by: Jens Oliver Meiert <jens@meiert.com>

### CODE REVIEW

File: pages/Documents/Reporting/OutboundReportingAPI/overview.md [hunk 1]
```diff
@@ -405,388 +409,397 @@ Click [**Transaction**](https://proactive-messaging.z1.fs.liveperson.com/api/api
...
 - Click on the user icon at top right corner and see the version.
 
 For Connect To Messaging:
-- Sign in to [this url](https://connect-to-messaging.fs.liveperson.com) or click on the quick launch icon from Conversation Cloud for Connect To Messaging.
+
+- Sign in to [Connect To Messaging](https://connect-to-messaging.fs.liveperson.com) or click on the quick launch icon from Conversation Cloud for Connect To Messaging.
 - Click on the user icon at top right corner and see the version.
 
-<strong>6. What is a LP data retention policy? And how long data is persisted for Outbound Reporting API?</strong>
+#### 6. What is a LP data retention policy? And how long data is persisted for Outbound Reporting API?
```
Comment (lkisoslive) [inline pages/Documents/Reporting/OutboundReportingAPI/overview.md:780]: What is a LP data retention policy? >>> either change to "an" or "the"

File: pages/Documents/Reporting/OutboundReportingAPI/overview.md [hunk 2]
```diff
@@ -405,388 +409,397 @@ Click [**Transaction**](https://proactive-messaging.z1.fs.liveperson.com/api/api
...
-<strong>8. What date range is supported by the Outbound Reporting API to pull raw transactional data?</strong>
+#### 8. What date range is supported by the Outbound Reporting API to pull raw transactional data?
 
-The caller can pull the data for a given 24 hour time interval from today or any previous day up to 13 months ago.
+The caller can pull the data for a given 24-hour time interval from today or any previous day up to 13 months ago.
 
-<strong>9. What date range is supported by the Outbound Reporting API to pull analytical data?</strong>
+#### 9. What date range is supported by the Outbound Reporting API to pull analytical data?
 
 The caller can pull the data for a given 60 day time interval from today or any previous day up to 13 months ago.
```
Comment (lkisoslive) [inline pages/Documents/Reporting/OutboundReportingAPI/overview.md:794]: 60 day time interval >> 60-day time interval

File: pages/Documents/Reporting/OutboundReportingAPI/overview.md [hunk 3]
```diff
@@ -405,388 +409,397 @@ Click [**Transaction**](https://proactive-messaging.z1.fs.liveperson.com/api/api
...
+
 - For Transaction API, pagination is supported. Page length is dynamic and varies from 1 to 9999.
 - For Analytics API, pagination is not needed.
 - For Proactive Campaign API, pagination is not supported yet.
 - For Proactive Campaign Analytics API, pagination is not needed.
 
-<strong>11. Is Outbound Reporting API real time? What is the delay in reporting data from the time Proactive Messaging campaigns or C2M deflections are created ?</strong>
-- Data in outbound reporting API can be delayed by up to 20 min. e.g. Proactive campaign created now will take up to 20 min to be reflected in the Reporting API. Similarly other messaging data like message delivered, opted out, conversation created etc will also take up to 20 min from the time the event occured.
\ No newline at end of file
+#### 11. Is Outbound Reporting API real time? What is the delay in reporting data from the time Proactive Messaging campaigns or C2M deflections are created?
```
Comment (lkisoslive) [inline pages/Documents/Reporting/OutboundReportingAPI/overview.md:803]: real time >> real-time

**Review by lkisoslive — APPROVED** [review]
Please consider my suggestions but otherwise looks good.

File: pages/Documents/Reporting/OutboundReportingAPI/overview.md [hunk 4]
```diff
@@ -227,175 +230,176 @@ Each of the filter options shown above are optional. Regardless of what filter o
...
 
-There are two versions of transaction API's.
+There are two versions of transaction APIs:
 
-| Version | Description  |
+| Version | Description |
 | :--- | :--- |
-| Version 2.0 (Recommended) | New version with simplified way for pagination and with additional filtering on message status and transaction id's along with channels and skills |
-| Version 1.0 (Deprecated) | Existing version of transaction API with filtering on channels and skills |
+| Version 2.0 (recommended) | New version with simplified way for pagination and with additional filtering on message status and transaction id's along with channels and skills |
```
[the remark this answers is not in the document]
Comment (j9t) [reply pages/Documents/Reporting/OutboundReportingAPI/overview.md:368]: Not a fan of this spelling, either (used to “ID”), but it’s common. Should be refactored in general.

Apostrophe looks wrong here, removed.

File: pages/Documents/Reporting/OutboundReportingAPI/overview.md [hunk 5]
```diff
@@ -405,388 +409,397 @@ Click [**Transaction**](https://proactive-messaging.z1.fs.liveperson.com/api/api
...
-<strong>3. What is the rate limit for the API’s?</strong>
+#### 3. What is the rate limit for the APIs?
 
 - Transaction API, rate: 10 TPS
 - Analytics API, rate: 10 TPS
 - Proactive Campaign API, rate: 10 TPS
 - Proactive Campaign Analytics API, rate: 10 TPS
 
-<strong>4. How can a brand find out which version of the Proactive Messaging or Connect To Messaging it uses ?</strong>
+#### 4. How can a brand find out which version of the Proactive Messaging or Connect To Messaging it uses ?
```
[the remark this answers is not in the document]
Comment (j9t) [reply pages/Documents/Reporting/OutboundReportingAPI/overview.md]: Had been done by an earlier PR.

File: pages/Documents/Reporting/OutboundReportingAPI/overview.md (hunk 1 above)
Comment (j9t) [reply pages/Documents/Reporting/OutboundReportingAPI/overview.md:780]: Changed it to “the LivePerson data retention policy”. Not a fan of the “LP” abbreviation in public docs—have replaced this in the past, might be good to take note on to refactor across the DC.

**Comment by j9t** [comment]
Thanks @lkisoslive! All feedback addressed.

**Pull request merged**